### PR TITLE
Fix managed vars filter

### LINKED ISSUE #3806: Managed variant filter next button resets filter..
After filtering with managed variants, the next page button clears filter and retrieves from the whole collection...

### PR DESCRIPTION
**Pull request by northwestwitch**
Fix #3806 

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the Pax software available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `cg-vm1`.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, repeat procedure at [https://pax.scilifelab.se/](https://pax.scilifelab.se), which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

<details>
<summary>Testing on hasta server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. `ssh <USER.NAME>@hasta.scilifelab.se`
1. Book your testing time using the Pax software. `us; paxa -u <user> -s hasta -r scout-stage`. You can also use the WSGI Pax app available at [https://pax.scilifelab.se/](https://pax.scilifelab.se).
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `conda activate S_scout; pip freeze | grep scout-browser`
1. Deploy the branch to test: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_scout -t scout -b <this_branch>`
1. Make sure the branch is deployed: `us; scout --version`
1. After testing is done, repeat the `paxa` procedure, which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

**How to test**:
1. Locally, go to Managed Variants page and use filter. Filter form should remember the selected options after the form is submitted.

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by DN
- [x] tests executed by CR

### CODE REVIEW

**Comment by northwestwitch** [comment]
This is ready for a review now!

File: scout/server/blueprints/managed_variants/templates/managed_variants/managed_variants.html [hunk 1]
```diff
@@ -400,6 +407,16 @@ <h5>
       });
       populateSelect(cytoStart, cytoStartOptions)
       populateSelect(cytoEnd, cytoStopOptions)
+      // Remember user query settings
+      selectedCytoStart =  "{{filters_form.cytoband_start.data|safe}}";
```
Comment (dnil) [inline scout/server/blueprints/managed_variants/templates/managed_variants/managed_variants.html:411]: 🥇

File: scout/server/blueprints/managed_variants/templates/managed_variants/managed_variants.html [hunk 2]
```diff
@@ -353,21 +354,27 @@ <h5>
...
+      chromSelect.value = selectedChrom;
+    }
     setCytobands(); // Reset cytobands select elements
   }
 
   function set_coords_fields(){
     //Make chromosome coordinates editable if a valid chromsome is selected
     selectedChrom = chromSelect.value;
-    if (selectedChrom !== 'undefined' && selectedChrom !== "All" ){
+    if (selectedChrom && selectedChrom !== "All" ){
```
Comment (dnil) [inline scout/server/blueprints/managed_variants/templates/managed_variants/managed_variants.html:368]: Looks good - guess that was from the previous select?!

**Review by dnil — APPROVED** [review]

File: scout/server/blueprints/managed_variants/templates/managed_variants/managed_variants.html (hunk 2 above)
Comment (northwestwitch) [reply scout/server/blueprints/managed_variants/templates/managed_variants/managed_variants.html:368]: Perhaps!

**Pull request merged**